[2023-04-13T20:19:14.633+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.inject_data_Kharkiv manual__2023-04-13T20:19:01.842128+00:00 [queued]>
[2023-04-13T20:19:14.762+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.inject_data_Kharkiv manual__2023-04-13T20:19:01.842128+00:00 [queued]>
[2023-04-13T20:19:14.765+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T20:19:14.770+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-13T20:19:14.772+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T20:19:14.836+0000] {taskinstance.py:1309} INFO - Executing <Task(SqliteOperator): inject_data_Kharkiv> on 2023-04-13 20:19:01.842128+00:00
[2023-04-13T20:19:14.844+0000] {standard_task_runner.py:55} INFO - Started process 6681 to run task
[2023-04-13T20:19:14.875+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'inject_data_Kharkiv', 'manual__2023-04-13T20:19:01.842128+00:00', '--job-id', '282', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmppmoiodri']
[2023-04-13T20:19:14.878+0000] {standard_task_runner.py:83} INFO - Job 282: Subtask inject_data_Kharkiv
[2023-04-13T20:19:15.047+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag.inject_data_Kharkiv manual__2023-04-13T20:19:01.842128+00:00 [running]> on host ffea11fb791c
[2023-04-13T20:19:15.407+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=inject_data_Kharkiv
AIRFLOW_CTX_EXECUTION_DATE=2023-04-13T20:19:01.842128+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-13T20:19:01.842128+00:00
[2023-04-13T20:19:15.411+0000] {sql.py:254} INFO - Executing: 
                INSERT INTO measures_ext (timestamp, temp, humidity, cloudiness, wind_speed) VALUES
                    (1681416975,
                    282.36,
                    91,
                    "overcast clouds",
                    3.41);
            
[2023-04-13T20:19:15.430+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T20:19:16.385+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T20:19:16.396+0000] {sql.py:375} INFO - Running statement: 
                INSERT INTO measures_ext (timestamp, temp, humidity, cloudiness, wind_speed) VALUES
                    (1681416975,
                    282.36,
                    91,
                    "overcast clouds",
                    3.41);
            , parameters: None
[2023-04-13T20:19:16.432+0000] {sql.py:384} INFO - Rows affected: 1
[2023-04-13T20:19:16.503+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=inject_data_Kharkiv, execution_date=20230413T201901, start_date=20230413T201914, end_date=20230413T201916
[2023-04-13T20:19:16.589+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-13T20:19:16.646+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
